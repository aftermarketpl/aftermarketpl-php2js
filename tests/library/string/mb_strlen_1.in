$a = "ąćę";
return mb_strlen($a);