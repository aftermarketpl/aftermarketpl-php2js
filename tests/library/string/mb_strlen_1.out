a = "\u0105\u0107\u0119";
return (a).length;
